<launch>
	<!-- global rosparam -->
	<param name="use_sim_time" value="true"/>

	<!-- rosbag -->
	<node pkg="rosbag" type="play" name="player" args="--clock $(env HOME)/rosbag/infant/dkan_outdoor/2019-01-13-15-46-58.bag -r 5 -s 60"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find pc_heightmap_ground_filter)/rviz_config/velodyne.rviz"/>

	<!-- sensor driver -->
	<include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/>

	<!-- main -->
	<node pkg="pc_heightmap_ground_filter" type="pc_heightmap_ground_filter" name="pc_heightmap_ground_filter" output="screen">
        <remap from="/point_cloud" to="/velodyne_points"/>
		<param name="meter_per_cell" type="double" value="0.5"/>
		<param name="cell_per_axis" type="int" value="100"/>
		<param name="min_obstacle_height" type="double" value="0.5"/>
	</node>
</launch>